packager/app/libcrypto_threading.cc

```diff
@@ -6,46 +6,19 @@
 
 #include "packager/app/libcrypto_threading.h"
 
-#include <openssl/thread.h>
-
+#include <glog/logging.h>
+#include <mbedtls/threading.h>
 #include <memory>
 
-#include "packager/base/logging.h"
-#include "packager/base/synchronization/lock.h"
-#include "packager/base/threading/platform_thread.h"
-
 namespace shaka {
 namespace media {
 
-namespace {
-
-std::unique_ptr<base::Lock[]> global_locks;
-
-void LockFunction(int mode, int n, const char* file, int line) {
-  VLOG(2) << "CryptoLock @ " << file << ":" << line;
-  if (mode & CRYPTO_LOCK)
-    global_locks[n].Acquire();
-  else
-    global_locks[n].Release();
-}
-
-void ThreadIdFunction(CRYPTO_THREADID* id) {
-  CRYPTO_THREADID_set_numeric(
-      id, static_cast<unsigned long>(base::PlatformThread::CurrentId()));
-}
-
-}  // namespace
-
 LibcryptoThreading::LibcryptoThreading() {
-  global_locks.reset(new base::Lock[CRYPTO_num_locks()]);
-  CRYPTO_THREADID_set_callback(ThreadIdFunction);
-  CRYPTO_set_locking_callback(LockFunction);
+  //    mbedtls_threading_set_alt();
```

Comment:
Do we need this class at all now?

Reply:
doesn't look like this is needed anymore, cleaning it up